cohttp/src/cookie.ml

```diff
@@ -135,11 +135,12 @@ module Set_cookie_hdr = struct
 
   (* TODO: check dupes+order *)
   let extract hdr =
+    let (=) = Http.Header.Private.caseless_equal in
```

Comment:
Can you inline this definition? It's confusing to read as `=` usually refers to polymorphic comparison